app/src/main/java/telenav/demo/app/utils/CategoryAndFiltersUtil.kt

```diff
@@ -26,15 +26,15 @@ object CategoryAndFiltersUtil {
 
     val hotCategoriesList = arrayListOf(
         HotCategory("Food", R.drawable.ic_food_color, R.drawable.ic_food, "226"),
-        HotCategory("Banks / ATMs", R.drawable.ic_atm_color, R.drawable.ic_atm, "374"),
-        HotCategory("Grocery", R.drawable.ic_grocery_color, R.drawable.ic_grocery, "221"),
-        HotCategory("Shopping", R.drawable.ic_shopping_color, R.drawable.ic_shopping, "4090"),
-        HotCategory("Coffee", R.drawable.ic_coffee_color, R.drawable.ic_coffee, "241"),
         HotCategory("Parking", R.drawable.ic_parking_color, R.drawable.ic_parking, "600"),
-        HotCategory("Hotels / Motels", R.drawable.ic_hotel_color, R.drawable.ic_hotel, "595"),
-        HotCategory("Attractions", R.drawable.ic_attraction_color, R.drawable.ic_attraction, "605"),
+        HotCategory("Coffee", R.drawable.ic_coffee_color, R.drawable.ic_coffee, "241"),
         HotCategory("Fuel", R.drawable.ic_gas_color, R.drawable.ic_gas, "811"),
-        HotCategory("Electric Vehicle Charge Station", R.drawable.ic_ev_color, R.drawable.ic_ev, "771"),
+        HotCategory("Shopping", R.drawable.ic_shopping_color, R.drawable.ic_shopping, "4090"),
+        HotCategory("Hotels", R.drawable.ic_hotel_color, R.drawable.ic_hotel, "595"),
+        HotCategory("Banks", R.drawable.ic_atm_color, R.drawable.ic_atm, "374"),
+        HotCategory("Attractions", R.drawable.ic_attraction_color, R.drawable.ic_attraction, "605"),
+        HotCategory("EV Charger", R.drawable.ic_ev_color, R.drawable.ic_ev, "771"),
+        HotCategory("Grocery", R.drawable.ic_grocery_color, R.drawable.ic_grocery, "221"),
         HotCategory("More", R.drawable.ic_more_color, R.drawable.ic_more, "")
```

Comment:
We should use category alias instead of number

Reply:
Thanks, will do that changes in the pr request related to categories.